server/src/main/scala/com/emaginalabs/kuronometer/server/config/KuronometerJacksonModule.scala

```diff
@@ -0,0 +1,35 @@
+package com.emaginalabs.kuronometer.server.config
+
+import com.emaginalabs.kuronometer.core.model.{BuildTool, Platform}
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind._
+import com.twitter.finatra.json.modules.FinatraJacksonModule
+
+object KuronometerJacksonModule extends FinatraJacksonModule {
+  override protected val propertyNamingStrategy: PropertyNamingStrategy = PropertyNamingStrategy.LOWER_CAMEL_CASE
+
+  override val additionalJacksonModules = Seq(
+    new com.fasterxml.jackson.databind.module.SimpleModule {
+      addDeserializer(classOf[Platform], PlatformDeserializer)
+      addSerializer(classOf[Platform], PlatformSerializer)
+
+      addDeserializer(classOf[BuildTool], BuildToolDeserializer)
+      addSerializer(classOf[BuildTool], BuildToolSerializer)
+    })
```

Comment:
I went for jackson because  Finatra is already using it. But I'd keep an eye on the parsing to add circe in case we need more complex deserialization. In this case jackson could be really messy, for sure.
I agree reflexion is evil and circe is more functional. But it also add several boilerplate.
However, I've no reason to avoid circe if you prefer this approach.